src/util/getAccessibleChildText.js

```diff
@@ -10,10 +10,14 @@ import isHiddenFromScreenReader from './isHiddenFromScreenReader';
  * Returns a new "standardized" string: all whitespace is collapsed to one space,
  * and the string is lowercase
  * @param {string} input
- * @returns lowercase, single-spaced, trimmed string
+ * @returns lowercase, single-spaced, punctuation-stripped, trimmed string
  */
 function standardizeSpaceAndCase(input: string): string {
-  return input.trim().replace(/\s\s+/g, ' ').toLowerCase();
+  return input
+    .trim()
+    .replace(/[,.?!;:]/g, '') // strip punctuation
```

Comment:
```suggestion
    .replace(/[,.?¿!‽¡;:]/g, '') // strip punctuation
```
we should probably catch a few others

Reply:
Definitely agreed with the commit suggestion;

> This seems like one of those bug fixes that might also be a breaking change

We haven't released this rule, so I'm thinking that there wouldn't be a breaking change? Or do you mean that we should add a toggle for user convenience?

Reply:
oh, ha, good point :-) in that case, let's do it